CHANGELOG.md

```diff
@@ -0,0 +1,11 @@
+# Change history for ui-oai-pmh
+
+## 1.1.0 (IN PROGRESS)
+
+* Updated to use Babel 7.
+* Updated `stripes` to v3.0.0.
+* Updated `eslint` to v6.2.1.
+
+## 1.0.0 (https://github.com/folio-org/ui-app-template/tree/v1.0.0) (2020-03-02)
```

Comment:
`ui-app-template` is not the name of this module. Moreover, I think it should be the `1.0.0` release in the progress.

Reply:
Fixed